### PR TITLE
Add Spanish translation for redis/overview.md

### PR DESCRIPTION
**Pull request by alemohamad**
Add Spanish translation for redis/overview.md

### CODE REVIEW

File: docs/redis/overview.es.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+// o
+
+let res = try await app.redis.send(command: "PING", with: ["hello"])
+print(res)
+// "hello"
+```
+
+## Modo Pub/Sub
+
+Redis admite la capacidad de entrar en un [modo "Pub/Sub"](https://redis.io/topics/pubsub) donde una conexión puede escuchar "canales" específicos y ejecutar cierres específicos cuando los canales suscritos publican un "mensaje" (algún valor de datos).
```
Comment (jacostaf10) [inline docs/redis/overview.es.md:143]:
```suggestion
Redis admite la capacidad de entrar en un [modo "Pub/Sub"](https://redis.io/topics/pubsub) donde una conexión puede escuchar "canales" específicos y ejecutar closures (métodos) específicos cuando los canales suscritos publican un "mensaje" (algún valor de datos).
```

File: docs/redis/overview.es.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+Hay un ciclo de vida definido para una suscripción:
+
+1. **subscribe**: invocado una vez cuando la suscripción comienza por primera vez
+2. **message**: invocado 0 o más veces a medida que se publican mensajes en los canales suscritos
+3. **unsubscribe**: invocado una vez cuando la suscripción termina, ya sea por solicitud o por pérdida de conexión
+
+Cuando creas una suscripción, debes proporcionar al menos un [`messageReceiver`](https://swiftpackageindex.com/swift-server/RediStack/main/documentation/redistack/redissubscriptionmessagereceiver) para manejar todos los mensajes que son publicados por el canal suscrito.
+
+Opcionalmente, puedes proporcionar un `RedisSubscriptionChangeHandler` para `onSubscribe` y `onUnsubscribe` para manejar sus respectivos eventos del ciclo de vida.
```
Comment (jacostaf10) [inline docs/redis/overview.es.md:153]:
```suggestion
Opcionalmente, puedes proporcionar un `RedisSubscriptionChangeHandler` para `onSubscribe` y `onUnsubscribe` que maneje sus respectivos eventos del ciclo de vida.
```

**Review by jacostaf10 — COMMENTED** [review]
Two minor suggestions to consider, the rest is all right.

**Comment by alemohamad** [comment]
Updated changes!

**Review by jacostaf10 — APPROVED** [review]
Ready!

**Pull request merged**